### ISSUE TITLE
FEAT: Query Results Ordering

### ISSUE THREAD

**Issue by cmwarre** [beekeeper-studio/beekeeper-studio#1077]
**Is your feature request related to a problem? Please describe.**

There are times that I write queries with variables declared at the top such as:

`
SET @rank=0;

SELECT
	@rank:=@rank+1 AS rank,
	t.*
FROM (...) t
`

I would expect the last "result" to be shown in the results section of the app but instead I get "Query Executed Successfully.  0 rows affected".  It took me a bit of time to discover the little "Result" selector combo box at the bottom so that I could actually see my data.  

**Describe the solution you'd like**
I'd generally expect a "stack" type interface where the last query to execute is shown first and then the 1st query to execute be shown last.  

Alternatively it might be good for the UI to be adjusted to show the "Results" combo box a bit more prominently or as the results window header?  

**Database type, if appropriate**
I was using MariaDB for this and haven't tested it if happens on other database flavors or not...

**Comment by rathboma** [comment]
Yes. I agree here! There shouldn't be an entry in the 'results' drop down if that query didn't produce any results.
